example/lib/widgets/initialize.dart

```diff
@@ -21,24 +21,37 @@ class _InitializeState extends BaseSampleWidgetState<Initialize> {
   Future<String> callDidomiPlugin() async {
     String languageCodeText = _languageController.text;
     String? languageCode = languageCodeText.isEmpty ? null : languageCodeText;
+
     String noticeIdText = _noticeIdController.text;
     String? noticeId = noticeIdText.isEmpty ? null : noticeIdText;
+
+    String tvNoticeIdText = _tvNoticeIdController.text;
+    String? tvNoticeId = tvNoticeIdText.isEmpty ? null : tvNoticeIdText;
+
     await DidomiSdk.initialize(
         _apiKeyController.text,
         disableDidomiRemoteConfig: _disableRemoteConfigValue,
         languageCode: languageCode,
-        noticeId: noticeId);
+        noticeId: noticeId,
+        tvNoticeId: tvNoticeId,
```

Comment:
Maybe the argument should be called androidTvNoticeId, as it is only used for Android